content/chainguard/libraries/javascript/build-configuration.md

```diff
@@ -216,6 +216,44 @@ To change the packages, remove the `node_modules` directory and the
 
 Now you can proceed with your development and testing. 
 
+### Minimal example project
+
+Use the following steps to create a minimal example project for pnpm with
```

Comment:
I swear the pnpm site is trying to annoy me specifically: https://pnpm.io/

Navigating there from google, I've seen it styled as `pnpm`, `PNPm`, and `pnPm`. I don't know why/how it would ever show up differently

Reply:
their docs go with `pnpm` so you're all set here, but why is it like this??

Reply:
I think they are just being funny on their website..